project/plugins.sbt

```diff
@@ -1,2 +1,12 @@
+resolvers += Opts.resolver.sonatypeSnapshots
+
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.6")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.2.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.11")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.2")
+addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.3.2")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.5")
+addSbtPlugin("io.github.jonas" % "sbt-paradox-material-theme" % "0.6.0")
+addSbtPlugin("com.drobisch" % "sbt-paradox-diagrams" % "0.0.2-SNAPSHOT")
```

Comment:
This is a bit strange that we "as a plugin" use an older version of "us" as a plugin.
Can't we somehow use the compiled code to generate the docs?
(It's not directly a question for you, I will dig into this, just don't want to forget.)

Reply:
https://stackoverflow.com/questions/9177923/sbt-plugin-that-depends-on-itself-to-build-itself this is probably it

Reply:
:+1: added the sources to the build project